src/compound/SupplyVaultBase.sol

```diff
@@ -9,6 +9,8 @@ import {IERC20, SafeERC20} from "@openzeppelin/contracts/token/ERC20/utils/SafeE
 import {CompoundMath} from "@morpho-labs/morpho-utils/math/CompoundMath.sol";
 import {Types} from "@contracts/compound/libraries/Types.sol";
 
+import "forge-std/console.sol";
```

Comment:
to remove

Reply:
I think it is a rebase issue, the first commit should not be there